src/Android/Xamarin.Android/Samples/Network Analysis/RouteAroundBarriers/readme.md

```diff
@@ -0,0 +1,66 @@
+# Route around barriers
+
+Find a route that reaches all stops without crossing any barriers.
+
+![Screenshot](RouteAroundBarriers.jpg)
+
+## Use case
+
+You can define barriers to avoid unsafe areas, for example flooded roads, when planning the most efficient route to evacuate a hurricane zone. You could also use this functionality to plan routes when you know an area will be inaccessible due to a community activity like a run or a market night.
+
+## How to use the sample
+
+Click 'Add stop' to add stops to the route. Click 'Add barrier' to add areas that can't be crossed by the route. Click 'Route' to find the route and display it.
+
+## How it works
+
+1. Create the route task by calling `RouteTask.CreateAsync(_serviceUrl)`.
+2. Get the default route parameters for the service by calling `_routeTask.CreateDefaultParametersAsync`.
+3. When the user adds a stop, add it to the route parameters.
+    1. Normalize the geometry; otherwise the route job would fail if the user included any stops over the international date line.
```

Comment:
More accurately, the 180th degree meridian. The International Date Line doesn't completely follow this meridian (also called the "anti-meridian").